libbeat/docs/processors-using.asciidoc

```diff
@@ -312,9 +312,10 @@ range:
 ===== `network`
 
 The `network` condition checks if the field is in a certain IP network range.
```

Comment:
I did a little rewording on this after realizing that the specification didn't explain the behavior when more than one field is used. Can you please apply this:

```
The `network` condition checks whether a field's value falls within a specified
IP network range. If multiple fields are provided, each field value must match
its corresponding network range. You can specify multiple network ranges for a
single field, and a match occurs if any one of the ranges matches. If the field
value is an array of IPs, it will match if any of the IPs fall within any of the
given ranges. Both IPv4 and IPv6 addresses are supported.

The network range may be specified using CIDR notation, like "192.0.2.0/24" or 
"2001:db8::/32", or by using one of these named ranges:
```